src/cljs/main/broadfcui/page/workspace/notebooks/notebooks.cljs

```diff
@@ -103,6 +103,8 @@
               [:div {}
                (style/create-form-label "Extension URI")
                [input/TextField {:ref "extensionURI" :autoFocus true :style {:width "100%"}}]
+               (style/create-form-label "User Provided Script URI")
```

Comment:
Is this name okay?

Reply:
I think it needs a bit more explanation. Does UI-land have tooltips or little question mark icons that you can click/hover for more info?